### DISCUSSION TITLE
Why I can't see blocks and transactions details?

### DISCUSSION THREAD

**Discussion by icemagno** [blockscout/blockscout#9363] [Support] [answered]
Everything seems to working very well, but when I click into a transaction or block to take more details, I have a blank page. I can see no errors in browser logs.

Backend: [v5.2.3-beta.+commit.ea55f5ca](https://github.com/blockscout/blockscout/commit/ea55f5ca)
Frontend: [v1.23.0](https://github.com/blockscout/frontend/tree/v1.23.0)

Any help?

![t1](https://github.com/blockscout/blockscout/assets/4127512/a18394e3-5c77-46d6-9768-419658ce0d75)
![t2](https://github.com/blockscout/blockscout/assets/4127512/f7c1967f-8cf1-4fb7-a954-97aacf21f51a)

**Comment by icemagno** [comment]
Note: By calling ```/api/v2/transactions/0x3e542491f818d30ab534c6c47b9b8b4ce61332dbd34628e82d55c17576800552``` I receive a JSON object with some data.

**Reply by vbaranov** [reply, marked as answer]
Please update frontend to [v1.23.1](https://github.com/blockscout/frontend/tree/v1.23.1) or set `NEXT_PUBLIC_NETWORK_RPC_URL` env variable.

**Reply by icemagno** [reply]
Uptating the image to "1.23.1" solved the problem. I see the changelog and is exactly what I need.

**Comment by icemagno** [comment]
This is my env :

What value I must set in ```EXT_PUBLIC_NETWORK_RPC_URL``` ?

```
NEXT_PUBLIC_API_HOST=10.5.112.132
NEXT_PUBLIC_API_PROTOCOL=http
NEXT_PUBLIC_STATS_API_HOST=http://10.5.112.132:8080
NEXT_PUBLIC_NETWORK_NAME=Awesome chain
NEXT_PUBLIC_NETWORK_SHORT_NAME=Awesome chain
NEXT_PUBLIC_NETWORK_ID=5
NEXT_PUBLIC_NETWORK_CURRENCY_NAME=Ether
NEXT_PUBLIC_NETWORK_CURRENCY_SYMBOL=ETH
NEXT_PUBLIC_NETWORK_CURRENCY_DECIMALS=18
NEXT_PUBLIC_API_BASE_PATH=/
NEXT_PUBLIC_APP_HOST=10.5.112.132
NEXT_PUBLIC_APP_PROTOCOL=http
NEXT_PUBLIC_HOMEPAGE_CHARTS=['daily_txs']
NEXT_PUBLIC_VISUALIZE_API_HOST=http://10.5.112.132:8081
NEXT_PUBLIC_IS_TESTNET=true
NEXT_PUBLIC_API_WEBSOCKET_PROTOCOL=ws
NEXT_PUBLIC_API_SPEC_URL=https://raw.githubusercontent.com/blockscout/blockscout-api-v2-swagger/main/swagger.yaml
```
